mrp_bom_simple_report/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2023 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin Dupont (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "MRP BoM Simple Report",
+    "summary": "Print simple report for your Bill of Materials",
+    "version": "16.0.1.0.0",
+    "category": "Manufacturing",
+    "author": "GRAP, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "GRAP, Odoo Community Association (OCA)",
    "maintainers": ["quentinDupont"],
```

it allows you to merge later PR that improves or fixes your module.